deepinv/models/bilateral.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import annotations
+import torch
+from torch import Tensor
+from .base import Denoiser
+
+
+class BilateralFilter(Denoiser):
+    r"""
+    Bilateral filter.
+
+    The bilateral filter as it was introduced in :footcite:t:`tomasi1998bilateral`
+    where each output pixel is a normalized weighted average of neighboring pixels in a spatial window.
+    The weights factor into a spatial kernel and a range kernel:
+
+        \hat{x}_i = (1 / W_i) * \sum_{j \in \omega(i)} k_d(i - j) k_r(x_i - x_j) x_j
+
+    with Gaussian spatial kernel k_d and Gaussian range kernel k_r.
+    The spatial standard deviation $\sigma_d$ controls how fast the kernel
```

Comment:
```suggestion
    The spatial standard deviation :math:`\sigma_d` controls how fast the kernel
```

The usual LaTeX syntax unfortunately isn't supported by sphinx / reStructuredText